### DISCUSSION TITLE
Friction factor in PINS module

### DISCUSSION THREAD

**Discussion by liangyu-psi** [idaholab/moose#19371] [Q&A Modules: Navier-Stokes] [answered]
How to set the friction factor when I use Finite Volume Incompressible Porous media Navier Stokes module?

Because this module only supports constant properties and friction factors. I set the darcy friction factor as 0.02. But the frictional pressure seems much smaller than the pressure drop calculated by hand. I don't know if the method I used is wrong.

Hope to get your anwser.

**Comment by liangyu-psi** [comment]
@GiudGiud Master Guid, please help.

**Comment by GiudGiud** [comment, marked as answer]
Hello
Yeah we keep the friction correlations in pronghorn, an EC-ed code. But they are not that complicated.
So the Darcy friction factor is supposed to have a mu/rho prefactor and then a coefficient. Are you sure the 0.2 amounts to those?

Please just call me Guillaume

Guillaume

**Reply by liangyu-psi** [reply]
Hello Guillaume
Thanks for your reply.
I check the momentum equation. It seems that _f_darcy=_f_*u_superficial/2/De, not mu/rho.
Does it right? I plan to multiply ‘u_superficial/2/De’ on the real fricition factor which is 0.02.

**Reply by liangyu-psi** [reply]
mu/rho is too small, so I guess it is not the correct prefactor .

**Reply by liangyu-psi** [reply]
In addition, I counted the friction pressure drop calculated by hand and the value calculated by MOOSE. They are exactly ‘u_superficial/2/De’ times apart.

**Reply by GiudGiud** [reply]
depends on the definition. 
In the kernel we have this:
```  if (_use_Darcy_friction_model)
    friction_term += (*_cL)(_current_elem)(_index)*_rho(_current_elem) * _u_functor(_current_elem) /
                     _eps(_current_elem);
```
so basically f (the ones you are inputting) * rho * u_interstitial

So just adjust your `f` to be 1/(2De) * old_f (* porosity maybe?)

**Reply by GiudGiud** [reply]
I'll go read Pronghorn manual to see how we got to our formulation and report back

**Reply by liangyu-psi** [reply]
Thank you so much Guillaume. Please check if it is old_f * u_superficial/(2De).

**Reply by GiudGiud** [reply]
So in MOOSE we write the friction term as the sum of a linear and quadratic term. You could be talking about a linear friction model since you want the Darcy friction coefficient? There's also a quadratic term, which uses a Forchheimer friction coefficient. 

The linear term is W rho V, and we use a correlation for W. Same for Forchheimer, and we actually place the velocity inside that coefficient (so that the term in the equation is quadratic). None of these two actually simplify to what you are usually. We use correlations developed for pebble beds. 

Your model looks like it's quadratic, maybe more along the lines of the Darcy Weisbach equation.
So you should simply add the velocity inside the definition of the coefficient.
